spec/models/note_spec.rb

```diff
@@ -0,0 +1,4 @@
+require 'rails_helper'
+
+RSpec.describe Note, type: :model do
+end
```

Comment:
If that's something we want, let's add it as a chore.  It seems useful, but maybe not a priority.